src/components/Overlay.tsx

```diff
@@ -17,13 +17,17 @@ const Overlay = ({
   className?: string
 }) => {
   const dialogRef = useRef<HTMLDialogElement>(null)
-  useHotkeys('esc', onClose)
+  useHotkeys('esc', () => {
+    onClose()
+  })
 
   useEffect(() => {
     if (isOpen) {
       dialogRef.current?.showModal()
+      document.body.style.overflow = 'hidden'
```

Comment:
Please test shorter window sizes, my main concern is the user won't be able to access search results if they're out of view.

Reply:
Great call. Looks like it still works because the overlay content itself is scrollable even when the content behind it is not, so we're still good.

https://github.com/user-attachments/assets/6073d33e-6718-41e5-99d4-efb664f290ce